### ISSUE TITLE
Don't throw error if the matched keyword is commented

### ISSUE THREAD

**Issue by borisguery** [borisguery/git-keywords-checker#1]
We may need to add a configuration to determine if we want to strictly match commented keywords.

``` php
// var_dump($dummy);
```

It may not be matched based on flag.
